pr_testing/test_multiple_prs.sh

```diff
@@ -1157,11 +1158,18 @@ if [ "X$BUILD_OK" = Xtrue -a "$RUN_TESTS" = "true" ]; then
     DO_GPU_TESTS=true
     mark_commit_status_all_prs 'unittests/gpu' 'pending' -u "${BUILD_URL}" -d "Waiting for tests to start"
   fi
+  if [ $(echo ${ENABLE_BOT_TESTS} | tr ',' ' ' | tr ' ' '\n' | grep '^HLT_P2_TIMING' | wc -l) -gt 0 ] ; then
```

Comment:
also check if `$CMSSW_RELEASE_BASE/src/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh` exists. If this script is missing e.g. old release cycles then we do not want to run this test